scripts/wallet.js

```diff
@@ -710,8 +710,21 @@ export class Wallet {
     async #transparentSync() {
         if (!this.isLoaded() || this.#isSynced) return;
         const cNet = getNetwork();
-        await cNet.getLatestTxs(this);
-        getEventEmitter().emit('transparent-sync-status-update', '', '', true);
+        let fSynced = false;
+        try {
+            await cNet.getLatestTxs(this);
+            fSynced = true;
+        } catch {
+            // If all Explorers are down, we'll just rely on the local TXDB and display a warning
```

Comment:
I think this is more serious: `getLatestTxs` failed, wallet is almost unusable, balance is wrong and any attempt to create a tx will likely fail.

Reply:
Shield is fully usable and transactable. Meanwhile current MPW will not even open/load if explorers are down (which they are down far too frequently).